--- tennix-1.1/tennix.desktop~	2011-02-10 12:39:39.000000000 +0100
+++ tennix-1.1/tennix.desktop	2011-02-11 21:24:57.633333337 +0100
@@ -1,9 +1,13 @@
 [Desktop Entry]
 Name=Tennix!
 GenericName=Tennis game
+GenericName[pl]=Gra w tenisa
 Comment=Play tennis against the computer or a friend
-Exec=tennix
+Comment[pl]=Zagraj w tenisa przeciwko komputerowi lub z przyjacielem
+Exec=tennix -f
 Icon=tennix
 Terminal=false
 Type=Application
 Categories=Game;SportsGame;
+Encoding=UTF-8
+# vi: encoding=utf-8
