--- tennix-0.3.0/makefile~	2007-06-12 22:47:13.000000000 +0200
+++ tennix-0.3.0/makefile	2007-06-14 01:51:02.000000000 +0200
@@ -20,6 +20,7 @@
 #
 
 CC        =  gcc
+CXXFLAGS  =
 RELEASE   =  svn
 
 TARGET ?= default
@@ -51,7 +52,7 @@
 endif
 
 tennix: $(OBJ)
-	$(CC) $(CFLAGS)   -o tennix $(OBJ) $(LIBS)
+	$(CC) $(CXXFLAGS) $(CFLAGS)   -o tennix $(OBJ) $(LIBS)
 	test -f tennix.exe && upx tennix.exe || true 
 
 tennix.o: tennix.c tennix.h game.h graphics.h input.h sound.h
