--- tennix-1.0/tennix.h~	2009-03-04 13:25:47.000000000 +0100
+++ tennix-1.0/tennix.h	2009-03-04 13:26:53.000000000 +0100
@@ -37,7 +37,7 @@
 #ifdef MACOSX
 #  define ARCHIVE_FILE_INSTALLED "Tennix.app/Contents/Resources/" ARCHIVE_FILE
 #else
-#  define ARCHIVE_FILE_INSTALLED "/" PREFIX "/share/games/tennix/" ARCHIVE_FILE
+#  define ARCHIVE_FILE_INSTALLED "/usr/share/tennix/tennix.tnx"
 #endif
 
 #ifdef WIN32
