--- tennix-0.4.1/makefile~	2007-09-24 22:04:45.000000000 +0200
+++ tennix-0.4.1/makefile	2007-09-24 22:04:43.000000000 +0200
@@ -28,6 +28,7 @@
 
 LIBS =
 CFLAGS += -Wall -DVERSION=\"$(RELEASE)\"
+LDFLAGS =
 
 ifeq ($(DEBUG),1)
   CFLAGS += -DDEBUG
@@ -53,7 +54,7 @@
 endif
 
 tennix: $(OBJ)
-	$(CC) $(CFLAGS)   -o tennix $(OBJ) $(LIBS)
+	$(CC) $(CFLAGS) $(LDFLAGS)   -o tennix $(OBJ) $(LIBS)
 	test -f tennix.exe && upx tennix.exe || true 
 
 install: tennix
