--- tennix-1.1/makefile~	2011-02-11 10:42:35.344805388 +0100
+++ tennix-1.1/makefile	2011-02-11 10:43:13.145319008 +0100
@@ -37,7 +37,7 @@
 DATADIR ?= $(DATAROOTDIR)/games
 
 LIBS =
-CFLAGS += -W -Wall -ansi -pedantic -Wcast-qual -Wwrite-strings -DVERSION=\"$(RELEASE)\" -O2 -DPREFIX=\"$(PREFIX)\" -g
+CFLAGS += -W -Wall -ansi -pedantic -Wcast-qual -Wwrite-strings -DVERSION=\"$(RELEASE)\" $(OPTFLAGS)
 CXXFLAGS += $(CFLAGS)
 
 USE_PYTHON ?= 1
